# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../log4cxx/log4cxx-0.10.0-headers-install.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Fix installation of header files

log4cxx_private.h and log4cxx.h are included in the globbing pattern used
for finding the headers, thus there is no need to include them explicitely
because `install` will refuse to install them twice.

--- apache-log4cxx-0.10.0/src/main/include/log4cxx/private/Makefile.am.orig	2011-04-17 17:55:26.216077822 +0200
+++ apache-log4cxx-0.10.0/src/main/include/log4cxx/private/Makefile.am	2011-04-17 17:55:39.172077859 +0200
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 privateincdir = $(includedir)/log4cxx/private
-privateinc_HEADERS= $(top_builddir)/src/main/include/log4cxx/private/*.h log4cxx_private.h
+privateinc_HEADERS= $(top_builddir)/src/main/include/log4cxx/private/*.h
 DISTCLEANFILES = log4cxx_private.h
 EXTRA_DIST = log4cxx_private.hw
 
--- apache-log4cxx-0.10.0/src/main/include/log4cxx/Makefile.am.orig	2011-04-17 18:04:46.688082284 +0200
+++ apache-log4cxx-0.10.0/src/main/include/log4cxx/Makefile.am	2011-04-17 18:05:16.932064300 +0200
@@ -15,7 +15,7 @@
 #
 SUBDIRS = helpers net nt rolling spi varia xml config db private pattern filter
 log4cxxincdir = $(includedir)/log4cxx
-log4cxxinc_HEADERS= $(top_srcdir)/src/main/include/log4cxx/*.h log4cxx.h
+log4cxxinc_HEADERS= $(top_srcdir)/src/main/include/log4cxx/*.h
 DISTCLEANFILES = log4cxx.h
 EXTRA_DIST = log4cxx.hw
 
